lib/screens/ol_crowding/dynamic_supervisor.ex

```diff
@@ -50,18 +44,6 @@ defmodule Screens.OlCrowding.DynamicSupervisor do
       restart: :transient
     }
 
-    case DynamicSupervisor.start_child(__MODULE__, spec) do
-      {:ok, child_pid} ->
-        _ = :timer.exit_after(10_000, child_pid, :kill)
-
-      {:ok, child_pid, _} ->
-        _ = :timer.exit_after(10_000, child_pid, :kill)
-
-      {:error, error} ->
-        Logger.error("crowding_dyn_supervisor_process_error #{inspect(error)}")
-
-      _ ->
-        Logger.warn("Something went wrong with starting the crowding dynamic supervisor process")
-    end
```

Comment:
Not anymore because the logger now has an internal shutdown mechanism (see [here](https://github.com/mbta/screens/blob/f91f869c60094883b8b0092c3d678fed22e211ca/lib/screens/ol_crowding/logger.ex#L17)). It felt cleaner to have the process shut itself down than have it be shut down from the outside.